ucm2/USB-Audio/Lenovo-ThinkStation-P620-Main-Headset.conf

```diff
@@ -0,0 +1,19 @@
+SectionDevice."Headphones" {
+	Comment "Headphones"
+
+	Value {
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId},1"
+		JackControl "Headphone - Output Jack"
+	}
+}
+
+SectionDevice."Microphone" {
```

Comment:
I think this should be either "Mic" (if internal) or "Headset" (if external), see [use-case.h](https://github.com/alsa-project/alsa-lib/blob/master/include/use-case.h#L112-L144)